ekf_odom:
  ros__parameters:
    use_sim_time: true
    frequency: 30.0
    two_d_mode: true

    map_frame: map
    odom_frame: odom
    base_link_frame: base_link
    world_frame: odom

    publish_tf: true

    # Fuse only position (x, y) and yaw from odometry
    odom0: /model/vehicle_blue/odometry
    odom0_config: [ true, true, false,
                    false, false, true,
                    false, false, false,
                    false, false, false,
                    false, false, false ]
    odom0_differential: false
    odom0_relative: false
